### PR TITLE
add tooltip prop and fix folder name

### LINKED ISSUE #22899: Add tooltip to SecretFormField

### PR DESCRIPTION
**Pull request by shavonn**
Fixes #22899

### CODE REVIEW

**Review by torkelo — APPROVED** [review]

File: packages/grafana-ui/src/components/SecretFormField/SecretFormField.tsx [hunk 1]
```diff
@@ -40,13 +41,15 @@ export const SecretFormField: FunctionComponent<Props> = ({
...
   isConfigured,
+  tooltip,
   placeholder = 'Password',
   ...inputProps
 }: Props) => {
   const styles = getSecretFormFieldStyles();
   return (
     <FormField
       label={label!}
+      tooltip={tooltip!}
```
Comment (ryantxu) [inline packages/grafana-ui/src/components/SecretFormField/SecretFormField.tsx]: the `!` means it can not be null... is that required here?

**Comment by ryantxu** [comment]
I don't see how this change makes the tests fail:
image

maybe merge with master again and try running test locally?

**Comment by shavonn** [comment]
> I don't see how this change makes the tests fail:
> image
> 
> maybe merge with master again and try running test locally?

@ryantxu it's the folder name. I fixed it in my PR.

File: packages/grafana-ui/src/components/SecretFormField/SecretFormField.tsx (hunk 1 above)
Comment (shavonn) [reply packages/grafana-ui/src/components/SecretFormField/SecretFormField.tsx]: @ryantxu I was following label but it's def not req. Will test some things out.

Comment (shavonn) [reply packages/grafana-ui/src/components/SecretFormField/SecretFormField.tsx]: @ryantxu so I tested it by replacing one of the if/else that needs to be replaced on Azure Monitor and it worked find. It doesn't seem to make it required.

**Review by dprokop — APPROVED** [review]

**Review by ryantxu — COMMENTED** [review]
LGTM

**Pull request merged**